From: Arturo Borrero Gonzalez <arturo@debian.org>
Subject: Allow nftables reproducible build
 This patch allows nftables to have reproducible build as seen by
 the service at reproducible.debian.net.
 .
 Currently, the only fix is for docbook2x which embeds the current date
 by default into generated manpages.
 .
 More info on this at:
 https://wiki.debian.org/ReproducibleBuilds/TimestampsInManpagesGeneratedByDocbook2x
Forwarded: not-needed
Last-Update: 2015-04-29

--- a/doc/Makefile.am
+++ b/doc/Makefile.am
@@ -8,11 +8,13 @@
 
 pdfdir=${docdir}/pdf
 
+BUILD_DATE=$(shell dpkg-parsechangelog -l../debian/changelog -S Date | LC_ALL=C date -u "+%d %B %Y" -f -)
+
 .xml.pdf:
 	${AM_V_GEN}dblatex -q -t pdf -o $@ $<
 
 .xml.8:
-	${AM_V_GEN}${DB2MAN} --xinclude $<
+	${AM_V_GEN}${DB2MAN} --string-param header-3="$(BUILD_DATE)" --xinclude $<
 
 EXTRA_DIST = nft.xml
 
